### ISSUE TITLE
ADISE module

### ISSUE THREAD

**Issue by rashivkp** [c4sc/ssa#7]
----------------------------------------
- Bitbucket: https://bitbucket.org/rashivkp/ssa/issue/7
- Originally reported by: Mohamed Rashid
- Originally created at: 2014-09-04T08:57:46.071

**Comment by rashivkp** [comment]
Issue #143 was marked as a duplicate of this issue.

----------------------------------------
Original comment by: Mohamed Rashid

**Comment by rashivkp** [comment]
Undefined variable: s (View: /home/rashi/public_html/ssa/app/views/backend/adise/student/edit.blade.php) (View: /home/rashi/public_html/ssa/app/views/backend/adise/student/edit.blade.php) 

----------------------------------------
Original comment by: Mohamed Rashid

**Comment by rashivkp** [comment]
@preethishinu  please merge develop/master branch on every feature branch you create. it should happen before reviewing. 

please fix the error and create a new pull request.

----------------------------------------
Original comment by: Mohamed Rashid

**Comment by rashivkp** [comment]
estimated-time: 1 day

----------------------------------------
Original comment by: Mohamed Rashid

**Comment by rashivkp** [comment]
ref #7. ADISE menu added


→ <<cset 39a99de8c083>>

----------------------------------------
Original comment by: Mohamed Rashid

**Comment by rashivkp** [comment]
ref #7. added ADISE module - forms for new student and list students


→ <<cset 6a2b335ec447>>

----------------------------------------
Original comment by: Mohamed Rashid

**Comment by rashivkp** [comment]
ref #7. ADISE edit and delete students


→ <<cset 1488df9889b4>>

----------------------------------------
Original comment by: Mohamed Rashid

**Comment by rashivkp** [comment]
ref #7. page for Basic student details added


→ <<cset 1b6186ab3ae4>>

----------------------------------------
Original comment by: Mohamed Rashid

**Comment by rashivkp** [comment]
ref #7. Filter student facilities


→ <<cset 621c5e2938e5>>

----------------------------------------
Original comment by: Mohamed Rashid

**Comment by rashivkp** [comment]
ref #7. ADISE module update corrected


→ <<cset 24759eefb977>>

----------------------------------------
Original comment by: Mohamed Rashid

**Comment by rashivkp** [comment]
ref #7. special code for Student generated


→ <<cset 6385feef047c>>

----------------------------------------
Original comment by: Mohamed Rashid

**Comment by rashivkp** [comment]
ref #7, #137

cleaning views, and changing routes.
remoteChained used for loading schools in combo.


→ <<cset 03a65ddd7724>>

----------------------------------------
Original comment by: Mohamed Rashid

**Comment by rashivkp** [comment]
ref #7

- updating composer
- cleaning views
- removing unnecessary routes, functions


→ <<cset 08d537d7dedd>>

----------------------------------------
Original comment by: Mohamed Rashid

**Comment by rashivkp** [comment]
fixed #7. correcting habitation column of students tables, and adding validations


→ <<cset 4180b65ad776>>

----------------------------------------
Original comment by: Mohamed Rashid

**Comment by rashivkp** [comment]
time-taken: 2 days

----------------------------------------
Original comment by: Mohamed Rashid
